packages/core/src/api.js

```diff
@@ -62,6 +62,12 @@ export function createPercyServer(percy, port) {
       build: percy.testing?.build ?? percy.build,
       loglevel: percy.loglevel(),
       config: percy.config,
+      widths: {
+        // This is always needed even if width is passed
+        mobile: percy.deviceDetails ? percy.deviceDetails.map((d) => d.width) : [],
```

Comment:
Shouldnt we send dpr as well ?

Reply:
doesn't make sense as we are just resizing page and not reloading page so changing dpr wouldn't help